core_gems.rb

```diff
@@ -11,4 +11,4 @@
 download "tzinfo", "1.2.2"
 download "tzinfo-data", "1.2016.5"
 download "google-protobuf", "3.5.1"
-download "grpc", "1.9.1"
```

Comment:
this `core_gems.rb` file also needs to be modified for the patch to work.

e.g.,

`s/download "grpc", "1.8.3"/download "grpc", "gem-fetch-1.8.3"/`

you could use this to also download the native `google-protobuf` package too (though it might not be necessary, since it looks like the `google-protobuf` package is building from source ok:

To do that for protobuf too: `s/download "google-protobuf", "3.5.1"/download "google-protobuf", "gem-fetch-3.5.1"`